uvloop/loop.pyx

```diff
@@ -1560,16 +1560,17 @@ cdef class Loop:
         transport.set_protocol(ssl_protocol)
         conmade_cb = self.call_soon(ssl_protocol.connection_made, transport)
         resume_cb = self.call_soon(transport.resume_reading)
+        app_transport = (<SSLProtocol>ssl_protocol)._app_transport
 
         try:
             await waiter
         except Exception:
-            transport.close()
+            app_transport.close()
```

Comment:
Wow, we need to backport this fix to CPython. Care to submit a PR?

Reply:
No problem, will do!